### PR TITLE
PXC-3217: wsrep_sst_auth: remove

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

**Review by rameshvs02 — COMMENTED** [review]
1) Replace `CentOS 6.8 servers` with `CentOS 7 servers`
2) Did not remove `wsrep_sst_auth` from node2 and node3 cnf code blocks. 
3) Change Galera library location in cnf code blocks. 
```
######## wsrep ###############
# Path to Galera library
wsrep_provider=/usr/lib64/galera4/libgalera_smm.so
```

4) Replace ;
```
It is not recommended to leave an empty password for the root account. Password can be changed as follows:

mysql@percona1> UPDATE mysql.user SET password=PASSWORD("Passw0rd") where user='root';
mysql@percona1> FLUSH PRIVILEGES;
```
with
```
Copy the automatically generated temporary password for the superuser account:

$ sudo grep 'temporary password' /var/log/mysqld.log
Use this password to log in as root:

$ mysql -u root -p
Change the password for the superuser account and log out. For example:

mysql> ALTER USER 'root'@'localhost' IDENTIFIED BY 'rootPass';
Query OK, 0 rows affected (0.00 sec)

mysql> exit
Bye
```

5) Replace `/etc/init.d/mysql start` with `systemctl start mysql`
6) Remove this line (SST script will create internal user for data transfer) `This means that the second node won't have empty root password anymore. In order to connect to the cluster and check the status, the root password from the first node should be used.`

File: doc/source/.res/replace.txt [hunk 1]
```diff
@@ -30,3 +30,16 @@
 
 .. |data-at-rest| replace:: *data at rest*
 .. |wsrep_state_dat| replace:: :file:`wsrep_state.dat`
+
+.. |pxc-sst-root| replace:: `mysql.pxc.sst.root`
+.. |mysql-pxc-sst-root| replace:: :term:`mysql.pxc.sst.root`
```
Comment (kennt-percona) [inline doc/source/.res/replace.txt]: the name has been changed
'mysql.pxc.sst.root' -> 'mysql.pxc.internal.session'

File: doc/source/glossary.rst [hunk 2]
```diff
@@ -299,3 +299,29 @@
 
       For each table, the server will create a file with the :file:`.frm`
       extension containing the table definition (for all storage engines).
+
+   mysql.pxc.sst.root
```
Comment (kennt-percona) [inline doc/source/glossary.rst]: As above, this has been renamed, to 'mysql.pxc.internal.session'

**Review by kennt-percona — APPROVED** [review]
lgtm

**Review by rameshvs02 — COMMENTED** [review]
lgtm

**Pull request merged**